editor/src/components/editor/actions/actions.tsx

```diff
@@ -2131,26 +2122,18 @@ export const UPDATE_FNS = {
       // FIXME: this is a bit unfortunate as this action should just do what its name suggests, without additional flags.
       // For now there's not much more that we can do since the action here can be (and is) evaluated also for transient states
       // (e.g. a `textEdit` mode after an `insertMode`) created with wildcard patches.
-      return clearDragState(editor, derived, false)
+      return editor
```

Comment:
I wonder if this action and `UPDATE_EDITOR_MODE` should be merged now 😬

Reply:
added a follow-up comment